src/display/canvas.js

```diff
@@ -511,7 +511,7 @@ var CanvasGraphics = (function CanvasGraphicsClosure() {
       // Grayscale, 1 bit per pixel (i.e. black-and-white).
       var srcLength = src.byteLength;
       var dest32 = HasCanvasTypedArraysCached.value ?
-        new Uint32Array(dest.buffer) : new Uint32ArrayView(dest);
+        new Uint32Array(dest.buffer) : new globalScope.Uint32ArrayView(dest);
```

Comment:
I want to remove this construct: let's add inherited from TypedArray three-args constructor for Uint32Array and use `new Uint32Array(dest.buffer, 0, dest.buffer.byteLength >> 2)` here. We need to add byteLength and buffer properties for `window.CanvasPixelArray` though. Hopefully we can rid of HasCanvasTypedArraysCached stuff by doing that.